src/briefcase-guide.rst

```diff
@@ -1,13 +1,12 @@
-******************************
 ODK Briefcase
-******************************
+================
 
-:dfn:`ODK Briefcase` is a Java application for fetching and pushing forms and their contents. It can be used to gather and aggregate data from ODK Collect when there is no internet connectivity or you don't want to set up an ODK Aggregate Server.
+:dfn:`ODK Briefcase` is an open source Java application used for fetching and pushing forms and their contents. It can be used to gather and aggregate data from ODK Collect when there is no internet connectivity or you don't want to set up an ODK Aggregate Server.
 
 Typical Usage
```

Comment:
A screenshot and language similar to Collect/Aggregate is important to add here.